target/linux/ramips/dts/omega2p.dts

```diff
@@ -0,0 +1,185 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/input/input.h>
```

Comment:
Please include dt-bindings/gpio/gpio.h here as well and use the GPIO_ACTIVE_HIGH/GPIO_ACTIVE_LOW gpio macros. Have a look at the F5D8235_V1.dts for an example.